python/ql/src/experimental/dataflow/internal/TaintTrackingPublic.qll

```diff
@@ -6,6 +6,8 @@
 private import python
 private import TaintTrackingPrivate
 private import experimental.dataflow.DataFlow
+// Need to import since frameworks can extend `AdditionalTaintStep`
```

Comment:
```suggestion
// Need to import since frameworks can extend `AdditionalTaintStep`
// If you implement a new framework, be sure to update `experimental.semmle.python.Frameworks`
// such that your code is imported there.
```

Reply:
My rationale for adding that comment was that we _who maintain the library code_ could easily remember why frameworks needed to be imported into this file.

The only time you would need to remember to add an import to `Frameworks.qll` is when merging support for a new framework/library on this repo. (that is, if you're modeling things yourself, you would add these things in `Customizations.qll`). We _could_ end up in a situation where we forgot to add a import to `Frameworks.qll`, but I don't think this comment will be the one to save us.